curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-030.md

```diff
@@ -1,26 +1,28 @@
 ---
-id: 6193f74b982e4c5449f51a47
+id: 61fda339eadcfd92a6812bed
 title: Step 30
 challengeType: 0
 dashedName: step-30
 ---
 
 # --description--
 
-To make your sheet stand out from the page, you'll want to give it a border. Give the `#sheet` selector a `padding` property set to `10px` to create space between the content and the border, then give it a `border` property set to `2px solid #d0d0d5`.
+Before you get too far in to your styling, you should make use of the `sr-only` class. You can use CSS to make elements with this class completely hidden from the visual page, but still be announced by screen readers.
+
+The `span[class~="sr-only"]` selector will select any `span` element whose `class` *includes* `sr-only`. Create that selector, and give it a `border` property set to `0`.
```

Comment:
This doesn't change anything visually as far as I could tell - I'm not sure why I'm setting the border to 0 when there isn't a border. Just pointing it out - doesn't necessarily need any changes.

Reply:
I'm pretty certain this is a fairly standard "hide the element" CSS snippet - in fact, even [Boostrap uses it](https://github.com/twbs/bootstrap/blob/main/scss/mixins/_visually-hidden.scss).

The properties might not all be necessary in this project, but IMHO it's good to teach.

Reply:
Added a little note for this.